app/admin/user.rb

```diff
@@ -2,63 +2,46 @@
   # See permitted parameters documentation:
   # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
   #
-  permit_params :email, :first_name, :last_name, :username
-  #
-  # or
-  #
-  # permit_params do
-  #   permitted = [:permitted, :attributes]
-  #   permitted << :other if resource.something?
-  #   permitted
-  # end
+  permit_params :email, :name, :image, :gender
 
   form do
     inputs 'Details' do
       input :email
-      input :first_name
-      input :last_name
-      input :username
+      input :name
+      input :gender
     end
-
     actions
   end
 
   index do
     selectable_column
     id_column
     column :email
-    column :first_name
-    column :last_name
-    column :username
-    column :sign_in_count
-    column :created_at
```

Comment:
This is a good column to have in the Admin User index page. Also the updated_at.